Project architecture/MVVM.md

```diff
@@ -50,6 +82,16 @@ The ViewModel together with all the data will survive the activity configuration
 
 Most applications contain some screens that can be grouped into a flow. Inside this flow, you will often have to share some data. Information from the previous screen might be needed or might influence some of the future screens. The ViewModel and LiveData can be the perfect solution for such situations. If you organize a specific flow inside one activity and all other screens as a fragment, it is easy to share a ViewModel among all fragments and the controlling activity. This makes it easy to share, store, combine, and manage all the data in the flow.
 
+## Coroutines vs RxJava
```

Comment:
Great chapter!
But should this really be under MVVM? Common Android?

Reply:
I agree, it is a bit out of place in the MVVM chapter, maybe we could move it to some other chapter as you suggested.